catkin_tools/verbs/catkin_build/build.py

```diff
@@ -732,12 +733,24 @@ def set_error_state(error_state):
                     # Print them in order of started number
                     for job_msg_args in sorted(executing_jobs, key=lambda args: args['number']):
                         msg += clr("[{name} - {run_time}] ").format(**job_msg_args)
-                    msg_rhs = clr("[{0}/{1} Active | {2}/{3} Completed]").format(
-                        len(executing_jobs),
-                        len(executors),
-                        len(packages) if no_deps else len(completed_packages),
-                        total_packages
-                    )
+
+                    if MakeJobServer.get_instance().supported:
+                        msg_rhs = clr("[{0}/{1} Jobs][{2}/{3} Packages][{4}/{5} Completed]").format(
+                            MakeJobServer.get_instance().num_running_jobs(),
+                            MakeJobServer.get_instance().num_jobs,
```

Comment:
Is there any reason to continue to get the instance? Are you expecting it to asynchronously change? If not it would be better to store it and reuse it to reduce function calls (which do cost in Python).